linux/step02_all_setup.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-if [ -z "$(getent passwd omero)" ]; then
+if [ -z "$(getent passwd omero-server)" ]; then
```

Comment:
Is an `omero` user still created anywhere? Or is that cleaning legacy check?

Reply:
an ``omero`` user is created in the parent Docker. 
This is an mistake when we split the omero user into ``omero-server`` and ``omero-web``